packages/sdk-core/src/common/TimeHelper.ts

```diff
@@ -0,0 +1,5 @@
+export class TimeHelper {
+    public static now(): number {
+        return Math.floor(new Date().getTime() / 1000);
+    }
```

Comment:
Can we make it obvious that this function is returning seconds and not milliseconds?